src/kaboom.ts

```diff
@@ -4084,7 +4114,18 @@ export default (gopt: KaboomOpt = {}): KaboomCtx => {
 				})
 			},
 
-			onHover(onHover: () => void, onNotHover: () => void): EventCanceller {
+			onHover(action: () => void): EventCanceller {
+				return this.onUpdate(() => {
+					if(this.isHovering()) {
+						if (this.hoverStarted) return
+						this.hoverStarted = true
+						this.hoverEnded = false
```

Comment:
These states should be initialized at the top of the object, and have type definition in types.ts under `AreaComp`